add-interactive.c

```diff
@@ -12,10 +12,12 @@ static int use_color = -1;
 
```

Comment:
I'm going to put a comment about commit message here, since I didn't find other appropriate place.

I would rename commit message to something like "built-in add -i: support ? (prompt help)"
to be more clear how `help` command and `?` are different.

Reply:
Good idea!